politeiawww/memorycache.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"sync"
+	"time"
+)
+
+// Item is a cached reference
+type Item struct {
+	Content    []byte
+	Expiration int64
```

Comment:
Is this a unix timestamp?  If so, make that apparent.

```suggestion
	Expiration int64  // Unix timestamp
```